packages/docs/src/app/manual-installation/page.md

```diff
@@ -66,7 +62,25 @@ export default defineConfig({
 });
 ```
 
-Next, create a `vitest.d.ts` file in the root of your project and add the following code:
+## Test Command Setup
+
+To have Poyro correctly run when `npm test` (or the test command for your package manager) is run this needs to be configured in `package.json`:
+
+```json
+{
+  "scripts": {
+    "test": "vitest",
+  }
+}
+```
+
+Note that any vitest CLI flags that you want to include by default when running `npm test` can be appended to the `vitest` command. For some useful ones see the [Troubleshooting](/#troubleshooting) section.
```

Comment:
I think this was better as a separate section for configuring vitest, since it really has nothing directly to do with poyro

Reply:
I see, but if you install manually the main guide won't work unless you set this up. I'm happy to call this out as vitest set up but I don't want our guide to be incomplete.